packages/dataviews/src/dataforms-layouts/panel/index.tsx

```diff
@@ -57,16 +59,45 @@ function DropdownHeader( {
 	);
 }
 
-function FormField< Item >( {
+function PanelDropdown< Item >( {
+	fieldDefinition,
+	popoverAnchor,
+	labelPosition = 'side',
 	data,
-	field,
 	onChange,
-}: FormFieldProps< Item > ) {
-	// Use internal state instead of a ref to make sure that the component
-	// re-renders when the popover's anchor updates.
-	const [ popoverAnchor, setPopoverAnchor ] = useState< HTMLElement | null >(
-		null
-	);
+	field,
+}: {
+	fieldDefinition: NormalizedField< Item >;
+	popoverAnchor: HTMLElement | null;
+	labelPosition: 'side' | 'top' | 'none';
+	data: Item;
+	onChange: ( value: any ) => void;
+	field: FormField;
+} ) {
+	const fieldLabel = isCombinedField( field )
+		? field.label
+		: fieldDefinition?.label;
+	const form = useMemo( () => {
+		if ( isCombinedField( field ) ) {
+			return {
+				type: 'regular' as const,
+				fields: field.children.map( ( child ) => {
+					if ( typeof child === 'string' ) {
+						return {
+							id: child,
+						};
+					}
```

Comment:
Currently we have this same logic in the `panel` and `regular` layouts, which is very similar to the `normalizeFormFields` function ( I believe I implemented this ). I think we could call that instead:
```
return normalizeFormFields( {
  type: 'regular' as const,
  fields: field.children
} );
```

This is something I can do as a follow up PR as this works fine as is.